packages/fxa-payments-server/src/lib/sentry.test.js

```diff
@@ -183,13 +182,4 @@ describe('lib/sentry', function() {
       assert.equal(resultUrl, expectedUrl);
     });
   });
-
-  describe('captureException', () => {
-    it('calls Sentry.captureException', () => {
-      jest.spyOn(Sentry, 'captureException');
-      sentryMetrics.captureException(new Error('testo'));
-      expect(Sentry.captureException).toHaveBeenCalled();
-      Sentry.captureException.mockRestore();
-    });
-  });
```

Comment:
idk, why this started triggering an error now but this was the error:

https://stackoverflow.com/questions/53162001/typeerror-during-jests-spyon-cannot-set-property-getrequest-of-object-which